### PR TITLE
Add use of MutationObserver to fix issues with new Google Calendar UI

### PR DESCRIPTION
**Pull request by apphancer**
This commit introduces the use of MutationObserver in order to address issues with Google's new Calendar UI.
MutationObserver observes for changes in the DOM and with this it's possible to apply the disable scroll function each time there is a relevant event change (e.g. changing month, week, calendar format, create a new event).

### CODE REVIEW

**Comment by DavGoodvibes** [comment]
How to try version #10 ? The one I tried was released by hamen as zip file : 
[google-calendar-scroll-disabler#9.zip](https://github.com/alter-ego/google-calendar-scroll-disabler/files/1404192/google-calendar-scroll-disabler.zip)

**Comment by apphancer** [comment]
Download my revised code from here https://github.com/apphancer/google-calendar-scroll-disabler
And install the extension locally

**Comment by DavGoodvibes** [comment]
apphancer : v0.3 is working for me. It look that you have your new version ;-)
Congratulations & thanks for your efforts.

**Comment by apphancer** [comment]
Thanks for your feedback @DavGoodvibes 

I have made a further commit where I have removed the jQuery library and changed 2 lines in content.js file lines to use VanillaJS

**Comment by x1c0** [comment]
Good job @apphancer! I was also trying to do this myself, first i also remove the use of jQquery but then I was struggling with the main grid changing and I could not find anything in the DOM that was changing.. the div that contains the main calendar is the same but after changing the month the listener was not firing.. 
Very clever approach with MutationObserver! I was not even aware that this existed 🙈 and i'm still not 100% sure how you do it with that `'data-viewfamily'` 😂 

I also fully tested this PR and it works perfectly👌 

Thank you!

File: code/content.js [hunk 1]
```diff
@@ -1,17 +1,38 @@
...
-    setTimeout(function () {
-
-        // Get a handle on the calendar grid
-        $('div[role="grid"]').on('mousewheel', function (e) {
-            
-            // Scrolling.... hahhahahaha I don't think so
+var disable_scroll = function () {
+    for (var live_selector of document.querySelectorAll(calendar_grid_selector)) {
+        live_selector.addEventListener('mousewheel', function (e) {
             if (e.target.id == 'el') return;
```
Comment (x1c0) [inline code/content.js:8]: Is this if condition still needed? I could not find any purpose for this, maybe this was needed on the previous Calendar version?

**Comment by danielosborn** [comment]
This branch/PR worked great for me (installed locally). Thank you for you work on this! Such an annoying "feature", I don't know why Google insists on keeping it or not having a way to turn it off.

**Pull request merged**